.github/workflows/ci.yaml

```diff
@@ -33,5 +33,5 @@ jobs:
       run: |
         echo "REGISTRY_USERNAME=${{ secrets.DOCKER_REGISTRY_USERNAME }}" >> $GITHUB_ENV
         echo "REGISTRY_PASSWORD=${{ secrets.DOCKER_REGISTRY_PASSWORD }}" >> $GITHUB_ENV
```

Comment:
I'm wondering if we should drop these - that way CI isn't pushing the image to docker; only build_rpms would do that.